modules/using-wildcard-routes.adoc

```diff
@@ -0,0 +1,317 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-controller
+
+[id="using-wildcard-routes_{context}"]
+= Using wildcard routes
+
+The HAProxy router has support for wildcard routes, which are enabled by setting
+the `ROUTER_ALLOW_WILDCARD_ROUTES` environment variable to `true`. Any routes
```

Comment:
This is no longer the case. https://github.com/openshift/enhancements/blob/master/enhancements/ingress/wildcard-admission-policy.md details how the wildcardPolicy is managed by the ingresscontroller.